lowdown.nix

```diff
@@ -0,0 +1,22 @@
+{ lib
```

Comment:
Could we move these *.nix files to their own subdirectory? The project root is now getting a bit cluttered.

Reply:
There is another package (change log d) in `misc` now